ra2ce/ra2ce_handler.py

```diff
@@ -60,9 +117,10 @@ def _initialize_logger(
         elif analysis:
             _output_config = AnalysisConfigData.get_data_output(analysis)
         else:
-            raise ValueError(
-                "No valid location provided to start logging. Either network or analysis are required."
+            logging.warning(
```

Comment:
You are also not initializing the ra2ce logger, please indicate so.

Reply:
Done